yellow-paper/docs/rollup-circuits/root_rollup.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Root Rollup
+sidebar_position: 4
+---
+
+The root rollup circuit is our top circuit, it applies the state changes passed through its children and the cross-chain messages. Essentially, it is the last step that allow us to prove that the state transition function $\mathcal{T}(S, B) \mapsto S'$ was applied correctly for a state $S$ and a block $B$. Note, that while its public inputs might not be the block itself, it have commitments to the. Provided with the block (`ProvenBlock`) and proof, a node is able to derive the public inputs from the block and his state and then execute the proof against the values derived. If the proof is passing, the block is valid and the state progression is correct.
```

Comment:
```suggestion
The root rollup circuit is our top circuit, it applies the state changes passed through its children and the cross-chain messages. Essentially, it is the last step that allows us to prove that the state transition function $\mathcal{T}(S, B) \mapsto S'$ was applied correctly for a state $S$ and a block $B$. Note, that the root rollup circuit's public inputs do not comprise all of the underlying data of a block, because the proof would be too costly to verify. Instead, the public inputs are commitments to the block's underlying data. Given a `ProvenBlock`(see below) a node can extract the root rollup's public inputs the block's state changes. Given the root rollup's proof and this extracted data, a node can verify the proof. If proof verifcation passes, the block is valid and the state progression is correct.
```

Perhaps provide a link to where $\mathcal{T}$ is defined, elsewhere in the yellow paper?